### PR TITLE
Added missing documentation for callback configuration

### PR DESCRIPTION
**Pull request by zephrax**
I needed to use the callback function, and found into the issues history that this feature actually exists but its not documented. So I've added the documentation to the README file.

Refs:
https://github.com/ahmadsoe/ember-highcharts/issues/19
https://github.com/ahmadsoe/ember-highcharts/pull/22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ ember install ember-highcharts
 This component takes in four arguments:
```
Comment (gregjopa) [inline README.md]: Can you update this to five arguments?

File: README.md [hunk 2]
```diff
@@ -57,6 +57,11 @@ Use this option to set the series data for your chart.
 The `theme` argument is optional and it allows you to pass in a
 [Highcharts theme](http://www.highcharts.com/docs/chart-design-and-style/themes).
 
+### callback
+
+The `callback` argument is executed after the chart object is finished loading and rendering.
```
Comment (gregjopa) [inline README.md]: Let's mention that its an optional parameter:

The `callback` argument is optional and is executed after the chart object is finished loading and rendering.

File: README.md [hunk 3]
```diff
@@ -94,12 +99,12 @@ export default Ember.Controller.extend({
...
 
   theme: defaultTheme
-
+  
 });
 ```
 
 ```handlebars
-{{high-charts chartOptions=chartOptions content=chartData theme=theme}}
+{{high-charts chartOptions=chartOptions content=chartData theme=theme callback=afterRenderCallback}}
```
Comment (gregjopa) [inline README.md]: I think the `callback` argument should be removed from this example. The `afterRenderCallback` doesn't exist in the example controller.

**Comment by nlfurniss** [comment]
Fixed in #197

**Pull request closed without merging**